lib/wallet/api/http/Cardano/Wallet/Launch/Cluster.hs

```diff
@@ -577,16 +577,16 @@ configurePool tr baseDir era metadataServer recipe = do
                   }
 
             let updateStaking sgs = sgs
-                    { Ledger.sgsPools = (Map.singleton poolId params)
+                    { Ledger.sgsPools = (ListMap.fromList [(poolId, params)])
```

Comment:
Just curious, is it possible to write:

```suggestion
                    { Ledger.sgsPools = (ListMap [(poolId, params)])
```

Reply:
I find the `fromList` version more obvious.